parsers/lib/web.py

```diff
@@ -13,6 +13,15 @@ def get_response(country_code, url, session=None):
     return response
 
 
+def get_response_with_params(country_code, url, session=None, params=None):
+    """Get response"""
+    ses = session or Session()
+    response = ses.get(url, params= params)
+    if response.status_code != 200:
+        raise ParserException(country_code, 'Response code: {0}'.format(response.status_code))
+    return response
+
```

Comment:
I can as well add an extra parameter in get_response method. Just did this so that it doesn't break anything which is already working.